worker-sys/src/service.rs

```diff
@@ -0,0 +1,14 @@
+use crate::Request;
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+extern "C" {
+    #[wasm_bindgen (extends = ::js_sys::Object, js_name = Fetcher)]
+    pub type Service;
```

Comment:
Now that dynamic dispatch is added, some of this boilerplate can be merged with [`Fetcher`](https://github.com/cloudflare/workers-rs/blob/main/worker-sys/src/fetcher.rs#L9).